File: solutions/go/raindrops/1/raindrops.go
package raindrops
import (
    "strings"
    "strconv"
)
func Convert(number int) string {
	var ans strings.Builder;
    if number % 3 == 0 {
        ans.WriteString("Pling")
    }
    if number % 5 == 0 {
        ans.WriteString("Plang")
    }
    if number % 7 == 0 {
        ans.WriteString("Plong")
    }

	if ans.Len() == 0 {
        return strconv.Itoa(number)
    }
    
	return ans.String()
}

File: solutions/cpp/raindrops/1/raindrops.cpp
#include "raindrops.h"

namespace raindrops {

std::string convert(int number) {
    std::string result;
    if (number % 3 == 0) {
        result += "Pling";
    }
    if (number % 5 == 0) {
        result += "Plang";
    }
    if (number % 7 == 0) {
        result += "Plong";
    }
    return result.empty() ? std::to_string(number) : result;
}

}  // namespace raindrops
